### PR TITLE
More on the Ext example file

### PR DESCRIPTION
**Pull request by adamtopaz**

### CODE REVIEW

File: src/examples/Ext.lean [hunk 1]
```diff
@@ -1,19 +1,66 @@
...
 noncomputable theory
 
 open category_theory category_theory.limits opposite
+open bounded_homotopy_category bounded_derived_category
+
+section Ext
+
+universes v u
+-- Let's work with an abelian category which has enough projectives.
+variables {A : Type u} [category.{v} A] [abelian A] [enough_projectives A]
```
Comment (jcommelin) [inline src/examples/Ext.lean:21]: I personally find it a tiny bit confusing if the category is called `A` and the second entry of `Ext(_, _)` is called `B`. My mind immediately thinks that `A` should be the first entry as well.
I guess the two obvious solutions are: use fancy curly letters for the category, or use `Y` for the snd entry.

File: src/examples/Ext.lean [hunk 2]
```diff
@@ -1,19 +1,66 @@
...
+
+/-- This is the (contravariant) delta functor `X ↦ Ext^*(X,B)`. -/
+example (B : A) : Aᵒᵖ ⥤δ Ab.{v} := Ext_δ_functor A B
+
+/-- The `n-th` component of this delta functor. -/
+example (n : ℕ) (X B : A) : Aᵒᵖ ⥤ Ab.{v} := Ext_δ_functor A B n
+example (n : ℕ) (X B : A) : Ext_δ_functor A B n = (Ext' n).flip.obj B := rfl
```
Comment (jcommelin) [inline src/examples/Ext.lean:28]: Should we evaluate both sides on an object, and get rid of the `flip`?

File: src/examples/Ext.lean [hunk 3]
```diff
@@ -1,19 +1,66 @@
...
+
+/-- `Ext' n (X,B) = Ext' n (X[0], B[0])`, functorially in `X`. -/
+example (n : ℕ) (X B : A) : (Ext' n).flip.obj B =
```
Comment (jcommelin) [inline src/examples/Ext.lean:31]: This isn't using the `X : A` assumption.

File: src/examples/Ext.lean [hunk 4]
```diff
@@ -1,19 +1,66 @@
...
+  (single _ 0).op ⋙ (Ext n).flip.obj ((single _ 0).obj B) := rfl
+
+/-- `Ext' 0 (-, B) ≅ Hom(-,B)` -/
+example (B : A) : (Ext' 0).flip.obj B ≅ preadditive_yoneda.obj B :=
```
Comment (jcommelin) [inline src/examples/Ext.lean:35]: Can we evaluation on objects, and make the RHS look as much as `Hom(X,Y)` as possible?

**Review by jcommelin — COMMENTED** [review]
We could add an example of a 5-term `exact_seq`.
I think the file is already quite convincing, but it might be nice to show how it's done.

File: src/examples/Ext.lean (hunk 4 above)
Comment (adamtopaz) [reply src/examples/Ext.lean]: done

File: src/examples/Ext.lean (hunk 3 above)
Comment (adamtopaz) [reply src/examples/Ext.lean]: done

File: src/examples/Ext.lean (hunk 2 above)
Comment (adamtopaz) [reply src/examples/Ext.lean]: done!

File: src/examples/Ext.lean (hunk 1 above)
Comment (adamtopaz) [reply src/examples/Ext.lean]: I used curly A's and changed the objects to X and/or Y.

**Review by jcommelin — APPROVED** [review]
LGTM!

**Pull request merged**